shell/browser/api/electron_api_screen.cc

```diff
@@ -126,6 +128,37 @@ void Screen::OnDisplayMetricsChanged(const display::Display& display,
                                 MetricsToArray(changed_metrics)));
 }
 
+gfx::Point Screen::ScreenToDIPPoint(const gfx::PointF& point_px) {
+#if BUILDFLAG(IS_WIN)
+  return display::win::ScreenWin::ScreenToDIPPoint(point_px);
+#elif defined(IS_OZONE_X11)
+  display::Display display =
+      GetDisplayNearestPoint(gfx::ToFlooredPoint(point_px));
+  gfx::Vector2d delta_px = point_px - display.native_origin();
+  gfx::Vector2dF delta_dip =
+      gfx::ScaleVector2d(delta_px, 1.0 / display.device_scale_factor());
+  return display.bounds().origin() + delta_dip;
+#else  // Wayland
+  return gfx::ToFlooredPoint(point_px);
+#endif
+}
+
+gfx::Point Screen::DIPToScreenPoint(const gfx::Point& point_dip) {
+#if BUILDFLAG(IS_WIN)
+  return display::win::ScreenWin::DIPToScreenPoint(point_dip);
+#elif defined(IS_OZONE_X11)
```

Comment:
For our build config this will be true for either backends on linux https://source.chromium.org/chromium/chromium/src/+/main:build/config/ozone.gni;l=83-87

We should have a runtime check instead
```suggestion
#elif defined(IS_LINUX)
if (ui::GetOzonePlatformName() == "x11") {
```

Reply:
Actually we have a better utility https://github.com/electron/electron/blob/9ec14b91e7e23f031e7763c95f2bdac4e20a0faf/shell/browser/native_window_views.cc#L166-L170, maybe move to some common utility file and reuse it.